room/compiler-processing/src/main/java/androidx/room/compiler/processing/javac/kotlin/KotlinMetadataElement.kt

```diff
@@ -54,6 +54,13 @@ internal class KotlinMetadataElement(
     }?.descriptor
 
     fun isObject(): Boolean = classMetadata.isObject()
+    fun isCompanionObject(): Boolean = classMetadata.isCompanionObject()
```

Comment:
seems like we'll read these a lot, would be good to cache the value (can be a followup CL)

Reply:
Sure, I can just do that now